specs/beacon-chain.md

```diff
@@ -510,32 +510,34 @@ Repeat while `slot - last_state_recalculation_slot >= CYCLE_LENGTH`:
 
 #### Adjust justified slots and crosslink status
 
-For all slots `s` in `last_state_recalculation_slot - CYCLE_LENGTH ... last_state_recalculation_slot - 1`:
+For every slot `s` in the range `last_state_recalculation_slot - CYCLE_LENGTH ... last_state_recalculation_slot - 1`:
 
-* Determine the total set of validators that attested to that block at least once
-* Determine the total balance of these validators. If this value times three equals or exceeds the total balance of all active validators times two, set `last_justified_slot = max(last_justified_slot, s)` and `justified_streak += 1`. Otherwise, set `justified_streak = 0`
-* If `justified_streak >= CYCLE_LENGTH + 1`, set `last_finalized_slot = max(last_finalized_slot, s - CYCLE_LENGTH - 1)`
+* Determine the total balance of validators that attested to the beacon chain block at slot `s`.
+* If the above total balance times three equals or exceeds the total balance of all active validators times two, set `last_justified_slot = max(last_justified_slot, s)` and `justified_streak += 1`. Otherwise, set `justified_streak = 0`.
```

Comment:
How about:

* Let `total_s_validator_balance` equal the total balance of validators that attested to the beacon chain block at slot `s`.
* If `3 * total_s_validator_balance >= 2* total_deposits`, set....

We could probably use a `total_deposits` helper too

Reply:
Yes, will do